index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="icon" type="image/png" href="../imgs/noun_51613_cc.png" aria-hidden="true">
+<meta name="author" content="Alessandro Lou">
+<meta name="description" content="Learn To Meditate">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>LEARN HOW TO MEDITATE!!!</title>
+
+    <!-- normalize across browsers -->
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+
+</head>
+
+<body>
+    <h1>Meditation</h1>
+    <img class="top-image" src="imgs/mandala.gif" alt="lotus" />
+
+    <nav>
+        <a href="#why">Why</a>
+        <a href="#vipassana">Vipassana</a>
+        <a href="#prepare">Prepare</a>
+        <a href="#sit">Sit</a>
+        <a href="#close">Close</a>
+        <a href="#focus">Breathe</a>
+        <a href="#letgo">Let</a>
+        <a href="#open">Open</a>
+    </nav>
+
+    <div class="extendednav">
```

Comment:
Redundant HTML object - should be restyled in your CSS based on screen size instead of being repeated and hidden